src/components/Movies/MovieCard.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { FC, useState } from 'react'
+import { Movie } from "../../interfaces";
+
+import styles from './MovieCard.scss'
+
+type MovieCardProps = Movie
+
+const MovieCard: FC<MovieCardProps> = ({
+  title,
+  poster_path,
```

Comment:
We usually use camelcase everywhere, why mix styles?

Reply:
I use snake case here because API response contains such property names. I'm not sure if we should modify them.

Reply:
All right, you used `{...item}` to spread the props, so let it be.